src/main/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/TurnoverPolicyServiceImpl.java

```diff
@@ -47,6 +57,39 @@ public TurnoverPolicy getTurnOverPolicy(String transactionId, String companyAcco
         return accountingPoliciesService
             .updateAccountingPoliciesApi(transactionId, companyAccountsId,
                 accountingPoliciesApi);
+    }
+
+    /**
+     * Validate the turnover policy details' length before calling the api. This string field should
+     * not be empty or contain empty characters.
+     *
+     * @param turnoverPolicy The turnover policy model that needs to be validated
+     * @return A list of validation errors. Or empty when turnoverPolicy contains valid information
+     */
+    private List<ValidationError> validateTurnoverPolicyDetails(TurnoverPolicy turnoverPolicy) {
+
+        List<ValidationError> validationErrors = new ArrayList<>();
+
+        if (turnoverPolicy.getIsIncludeTurnoverSelected() &&
+            IsRequiredFieldEmpty(turnoverPolicy.getTurnoverPolicyDetails())) {
+
+            ValidationError error = new ValidationError();
+            error.setFieldPath(TURNOVER_POLICY_DETAILS_FIELD_PATH);
+            error.setMessageKey(INVALID_STRING_SIZE_ERROR_MESSAGE);
+            validationErrors.add(error);
+        }
+
+        return validationErrors;
+    }
 
+    /**
+     * Check if field contains invalid information when it is not null: empty or contain empty
+     * characters.
+     *
+     * @param turnoverPolicyDetails the string field that needs to be assessed
+     * @return
+     */
+    private boolean IsRequiredFieldEmpty(String turnoverPolicyDetails) {
```

Comment:
This method should begin with a lower case 'i'. Plus, is it worth grouping the `turnoverPolicy.getIsIncludeTurnoverSelected()` check in this method? Alternatively, just remove this method and bundle the empty check in the `if` check above.

Reply:
i moved this `turnoverPolicy.getIsIncludeTurnoverSelected()` to the method and rename method to make it more descriptive